label_studio/core/label_config.py

```diff
@@ -84,7 +84,7 @@ def parse_config_to_json(config_string):
     if xml is None:
         raise etree.ParseError('xml is empty or incorrect')
     config = xmljson.badgerfish.data(xml)
-    config = _fix_choices(config)
+    #  config = _fix_choices(config)
```

Comment:
Is this intentionally left in? Why don't we want to `_fix_choices` anymore?

Reply:
nevermind, I see this was mentioned in the description - my feeling is that if we really don't need this we can delete the line (?)